Improvements to RPM installer

petebrew opened the pull request.

petebrew committed:
Renaming configure script

petebrew committed:
Improvements to RPM Install

petebrew committed:
Added missing Fedora commons configurations

petebrew committed:
Further improvements to RPM install

petebrew committed:
Improvements to webui RPM install

petebrew committed:
Use unhashed pwd in dccd.properties file

petebrew committed:
Merge branch 'master' of https://github.com/petebrew/dccd-webui.git

petebrew committed:
Merge branch 'master' of https://github.com/DANS-KNAW/dccd-webui.git

petebrew committed:
First attempt at adding charts to 'about' page

petebrew committed:
Python scripts for generating simple stats information which can then be
displayed on the website.

petebrew committed:
Python script for enumerating the number of objects in DCCD by
organisation

petebrew committed:
Basic stats charts displayed only when users are logged in.

petebrew committed:
Upgrading dependencies where no conflict exists

petebrew committed:
Merge branch 'master' of https://github.com/petebrew/dccd-webui.git

petebrew commented:
Contains lots of work on the RPM install method and upgrades to dependencies where there are no problems.

PaulBoon commented:
Could you remove all the work on the About page; stats, python scripts, etc. and make a separate pull request for it.  
There are some issues with it, which I want to deal with in a separate pull request and first merge the other stuff so we can go on.

petebrew committed:
Removing stats panel changes

As requested, I'm removing all changes associated with the stats panel
on the about page so that Paul can finish the pull request for the RPM
changes.

petebrew commented:
Stats panel on about page removed as requested.  I forgot these changes were in there I wrote them so long ago. They are not really ready for prime time because of the issue of needing users to be logged in to legitimately get access to this summary data.
